### PR TITLE
feature: Release NPM for PR

### PR DESCRIPTION
**Pull request by kevinshahfws**

### CODE REVIEW

File: .github/workflows/release-github.yml [hunk 1]
```diff
@@ -69,18 +70,25 @@ jobs:
         run: npx semantic-release --dry-run
       - name: Build artifacts for PRs
         if: github.event_name == 'pull_request'
-        run: npm run dist
+        run: |
+          npm version prerelease --preid pr207 --workspaces --git-tag-version false
```
Comment (kpears201) [inline .github/workflows/release-github.yml:74]: you don't need this twice now, --include-workspace-root fixes that

**Review by kpears201 — APPROVED** [review]

File: .github/workflows/release-github.yml [hunk 2]
```diff
@@ -35,41 +35,45 @@ jobs:
...
+          echo "NPM_DIST_TAG=next" >> $GITHUB_ENV
+          echo "DOCS_TAG=next" >> $GITHUB_ENV
+      - name: Set feature branch tags
+        if: github.ref_name != 'main' && github.ref_name != 'next' && github.event_name != 'pull_request'
         run: |
-          echo "Setting RELEASE_CHANNEL to '$(node ./src/js/version.mjs channel $NPM_DIST_TAG)' using branch name '$NPM_DIST_TAG'"
-          echo "RELEASE_CHANNEL=$(node ./src/js/version.mjs channel $NPM_DIST_TAG)" >> $GITHUB_ENV
-      - name: Override release channel for PRs
-        if: github.event_name == 'pull_request'
+          echo "NPM_DIST_TAG=test" >> $GITHUB_ENV
```
Comment (jlacivita) [inline .github/workflows/release-github.yml:51]: Won't this set the npm release channel to `test`?

**Review by jlacivita — APPROVED** [review]

**Pull request merged**